### PR TITLE
Pardi 202

### PR DESCRIPTION
**Pull request by UnixJunkie**
compatible with the latest version of the dolog library

### CODE REVIEW

**Comment by mseri** [comment]
Compilation fails on ocaml 4.05 and 4.06 due to 

```
#=== ERROR while compiling pardi.2.0.1 ========================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.05.0 | pinned(https://github.com/UnixJunkie/pardi/archive/v2.0.1.tar.gz)
# path                 ~/.opam/4.05/.opam-switch/build/pardi.2.0.1
# command              ~/.opam/4.05/bin/dune build -p pardi -j 2
# exit-code            1
# env-file             ~/.opam/log/pardi-23-5137e7.env
# output-file          ~/.opam/log/pardi-23-5137e7.out
### output ###
#       ocamlc src/.pardi.eobjs/byte/utls.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.05/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.pardi.eobjs/byte -I /home/opam/.opam/4.05/lib/batteries -I /home/opam/.opam/4.05/lib/bytes -I /home/opam/.opam/4.05/lib/cryptokit -I /home/opam/.opam/4.05/lib/ctypes -I /home/opam/.opam/4.05/lib/dolog -I /home/opam/.opam/4.05/lib/equeue -I /home/opam/.opam/4.05/lib/integers -I /home/opam/.opam/4.05/lib/lz4 -I /home/opam/.opam/4.05/lib/minicli -I /home/opam/.opam/4.05/lib/netcamlbox -I /home/opam/.opam/4.05/lib/netmulticore -I /home/opam/.opam/4.05/lib/netplex -I /home/opam/.opam/4.05/lib/netstring -I /home/opam/.opam/4.05/lib/netsys -I /home/opam/.opam/4.05/lib/num -I /home/opam/.opam/4.05/lib/ocaml/threads -I /home/opam/.opam/4.05/lib/parany -I /home/opam/.opam/4.05/lib/rpc -I /home/opam/.opam/4.05/lib/setcore -I /home/opam/.opam/4.05/lib/stdint -I /home/opam/.opam/4.05/lib/zarith -I /home/opam/.opam/4.05/lib/zmq -I src/.sec_chan.objs/byte -I src/.sec_chan.objs/native -no-alias-deps -o src/.pardi.eobjs/byte/utls.cmo -c -impl src/utls.ml)
# File "src/utls.ml", line 418, characters 2-7:
# Error: Unbound module Float
```

This could be easily fixed by increasing the lower bound on the compiler or using `stdlib-shims` to add the Float module.

**Comment by UnixJunkie** [comment]
Ok, I'll correct that.

**Comment by UnixJunkie** [comment]
I retagged then.

**Comment by mseri** [comment]
Thanks!

File: packages/pardi/pardi.2.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://github.com/UnixJunkie/pardi"
+bug-reports: "https://github.com/UnixJunkie/pardi/issues"
+dev-repo: "git+https://github.com/UnixJunkie/pardi.git"
+license: "GPL-1.0-or-later"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "dune" {>= "1.6"}
```
Comment (kit-ty-kate) [inline packages/pardi/pardi.2.0.2/opam]:
```suggestion
  "dune" {>= "1.11"}
```
This constraint is wrong according to your `dune-project` file. Could you fix this?

Comment (mseri) [reply packages/pardi/pardi.2.0.2/opam]: Oh sorry, I thought I had checked it. We should start making this check automatic... I'll send a PR

Comment (UnixJunkie) [reply packages/pardi/pardi.2.0.2/opam]: thank you both

**Pull request merged**